integrated_channels/api/v1/blackboard/serializers.py

```diff
@@ -18,13 +18,18 @@ class Meta:
         extra_fields = (
             'client_id',
             'client_secret',
```

Comment:
can we remove these two fields from here?

Reply:
@sameenfatima78 i planned to create a separate PR for removing these and feature flag, like done for other integrated channels